server/pkg/provision/auth.go

```diff
@@ -43,3 +61,119 @@ func emailSettings(ctx context.Context, s store.Storer) error {
 		})
 	})
 }
+
+// addAuthSuperUsers assigns BYPASS roles to users from AUTH_PROVISION_SUPER_USER value
+// When in Production, Corteza should stop and report an error.
+func addAuthSuperUsers(ctx context.Context, log *zap.Logger, s store.Storer, envOpt options.EnvironmentOpt) (err error) {
+	var (
+		val, has = os.LookupEnv("AUTH_PROVISION_SUPER_USER")
+	)
+
+	if envOpt.IsProduction() {
+		panic(errors.New(0, "Cannot provision super users in Production enviroment!"))
+	}
+
+	if has {
```

Comment:
Instead of saying

```
if something {
    some complex code
}
```

you can (should!!) do

```
if !something {
  return
}
some complex code
```

"Guard statements" let you flatten out the code which makes it more readable, safer, ...